src/explorer/WebAppTreeItem.ts

```diff
@@ -29,6 +26,9 @@ export class WebAppTreeItem extends SiteTreeItem {
     public get iconPath(): IThemedIconPath {
         return getThemedIconPath('WebApp_color');
     }
+    public static contextValue: string = extensionPrefix;
```

Comment:
Did you mean to move these?

Reply:
No, I wonder if it was some auto-formatting shenanigans going on...